plugins/birthday/cdbirthdaycalendar.cpp

```diff
@@ -184,9 +200,6 @@ void CDBirthdayCalendar::updateBirthday(const QContact &contact)
             qCWarning(lcContactsd) << Q_FUNC_INFO << "Failed to add event to calendar";
             return;
         }
-    } else {
-        // Update the existing event.
-        event->setReadOnly(false);
```

Comment:
Well, actually here it's the event that is read-only, not the notebook. Looking at KCalendarCore::Incidence::setSummary() for instance, it's a no-op when the event is read-only. So to make updates, the event should be set read-write temporarily.

Reply:
Ah darn, right. Suppose the test contact I used for different summary variants worked because it didn't have an earlier event. Does now have a double protection of readonly event + notebook, but maybe I'll just remove these lines from the changes.